src/status_im/chat/screen.cljs

```diff
@@ -20,6 +20,7 @@
             [status-im.chat.views.input.input :as input]
             [status-im.chat.views.toolbar-content :as toolbar-content]
             [status-im.ui.components.animation :as animation]
+            [status-im.ui.components.label :as l]
```

Comment:
On one hand, we shouldn't break the convention to use the last segment of the namespace as the alias. On the other hand `label/label` looks clumsy. 
How about we put this in `react` namespace and call it `react/label`. I mean, it's that kind of a low level abstraction.
Any thoughts @flexsurfer @jeluard ?

Reply:
i'm not sure, because in `react` it means, that it's react native library component, but it's not, hm, btw it's not only this place there it looks clumsy, so we need to find common solution for this case

Reply:
It wouldn't be the only "slightly higher lever component" than just included from react or a library. e.g. there is `icon` as a slight abstraction over `image`.